package.json

```diff
@@ -20,12 +21,14 @@
     "grunt-cli": "~0.1.13",
     "grunt": "~0.4.2",
     "grunt-contrib-connect": "~0.6.0",
-    "grunt-contrib-jshint": "~0.8.0",
+    "grunt-contrib-jshint": "~1.1.0",
     "chai": "~1.8.1",
     "mocha": "~1.17.1",
     "grunt-mocha-phantomjs": "~0.4.0",
-    "browserify": "~12.0.1",
-    "grunt-browserify": "~4.0.1",
+    "rollup": "^0.41.4",
+    "rollup-plugin-commonjs": "^7.0.0",
+    "rollup-plugin-node-resolve": "^2.0.0",
```

Comment:
Actually this can be dropped completely, see d940807db509a284756279f9c61b03db54f48542.